bundles/de.uka.ipd.sdq.simulation.abstractsimengine/src/de/uka/ipd/sdq/simulation/abstractsimengine/SimCondition.java

```diff
@@ -14,6 +14,7 @@
  * @param <M>
  *            the type of the simulation model
  */
+@FunctionalInterface
 public interface SimCondition {

```

Comment:
Any reason why we don't have an I in the name?

Reply:
I can't tell you. We are very inconsistent in this.